### PR TITLE
Dry up menu and separate links with pipe

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by ares** [comment]
menu

the menu items are separated by pipe `|` for better readability, also help page now uses the same helper instead of duplicating the code

File: app/helpers/apipie_dsl_helper.rb [hunk 1]
```diff
@@ -12,6 +12,23 @@ def heading(title, level = 1)
     end
   end
 
+  def apipie_dsl_menu
+    first = true
```
Comment (ofedoren) [inline app/helpers/apipie_dsl_helper.rb:16]: What's that for?..

Comment (ares) [reply app/helpers/apipie_dsl_helper.rb]: to test your attention :-) nah, I original had a different implementation and this is a leftover :-) dropped now

File: app/helpers/apipie_dsl_helper.rb [hunk 2]
```diff
@@ -12,6 +12,22 @@ def heading(title, level = 1)
     end
   end
 
+  def apipie_dsl_menu
+    content_tag(:ul, class: 'breadcrumb') do
+      content =  dsl_sections.map do |section|
```
Comment (ofedoren) [inline app/helpers/apipie_dsl_helper.rb:17]: I know it's an irrelevant thingie, but redundant space after `=`.

**Comment by ares** [comment]
Thanks, rebased and fixed

**Pull request merged**